doc/DTABASE_BACKUP_RESTORE.md

```diff
@@ -0,0 +1,150 @@
+
+# Database Backup and Restore
+
+> This repository provides a solution for backing up and restoring databases. Whether you need scheduled backups for automated routine tasks or manual backups for specific events, this guide will walk you through the process.
+
+## Backup Types
+
+### Scheduled Backups
+
+Scheduled backups are automated processes that run at predefined intervals. They help ensure that your data is regularly backed up without manual intervention.
+
+To set up scheduled backups, follow the steps outlined in the Configuration section.
+
+* Support: PostgreSQL
+
+### Manual Backups
+
+Manual backups are initiated by a user on-demand. These are useful for creating backups before system updates, major changes, or any event that requires a snapshot of the database at a specific point in time.
+
+To manually create a backup, follow the steps outlined in the Usage section.
+
+* Support: PostgreSQL, MySQL
+
+### Restore Process
+
+The restore process allows you to recover your database from a previously created backup.
+
+* Support: PostgreSQL, MySQL
+
+# CLI Operations
+
+## Create MySQL and PostgreSQL database
+
+```bash
+❯ ./civo --config staging.civo.json database create postgres-demo --size g3.db.medium --software PostgreSQL --version 14
```

Comment:
Can we change it to `civo database`. Basically, remove `./` and `--config` . So users can copy-paste commands

Reply:
Also, make a section in README.md to point to this doc. 
We have sections for resources. I think it's good to have separate docs but we should just link it from README
https://github.com/civo/cli?tab=readme-ov-file#region

Reply:
Addressed